src/main/java/seedu/address/logic/commands/LogCommand.java

```diff
@@ -0,0 +1,133 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_HOURS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.amenity.Amenity;
+import seedu.address.model.studyspot.Address;
+import seedu.address.model.studyspot.Favourite;
+import seedu.address.model.studyspot.Name;
+import seedu.address.model.studyspot.OperatingHours;
+import seedu.address.model.studyspot.Rating;
+import seedu.address.model.studyspot.StudiedHours;
+import seedu.address.model.studyspot.StudySpot;
+import seedu.address.model.tag.Tag;
+
+
+/**
+ * Deals with commands that change the number of hours studied at a location.
+ */
+public class LogCommand extends Command {
+    public static final String COMMAND_WORD = "log";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds to the studied hours at the "
+            + "study spot identified by its name (case-insensitive).\n "
+            + "If -o is added, it will set the studied hours to the value provided\n"
+            + "If -r is added, regardless of what value is provided, the value will be reset to 0\n"
```

Comment:
Might be clearer to mention the mnemonic used, so it's easier to remember what each flag is for.
e.g. "The -o flag will override the studied hours with the value provided"
e.g. "The -r flag will reset studied hours to 0"